### PR TITLE
feat: add devx balances and validators

### PR DESCRIPTION
**Pull request by mazzy89**
This PR adds:

- Iker Gno key address
- DevX Faucets addresses
- DevX Validators

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: misc/deployments/test4.gno.land/genesis.json [hunk 1]
```diff
@@ -15,6 +15,26 @@
...
+      "name": "devx-val-1"
+    },
+    {
+      "address": "g1t9ctfa468hn6czff8kazw08crazehcxaqa2uaa",
+      "pub_key": {
+        "@type": "/tm.PubKeyEd25519",
+        "value": "wBqj3F9RgIJ12UqGq5okzHfVbbd6H5AF5y4as8Ovx00="
+      },
+      "power": "",
+      "name": "devx-val-2"
```
Comment (mazzy89) [inline misc/deployments/test4.gno.land/genesis.json:35]: do we want to enforce any moniker conventions?

File: misc/deployments/test4.gno.land/genesis.json [hunk 2]
```diff
@@ -15,6 +15,26 @@
...
     }
   },
+  "validators": [
+    {
+      "address": "g1mxguhd5zacar64txhfm0v7hhtph5wur5hx86vs",
+      "pub_key": {
+        "@type": "/tm.PubKeyEd25519",
+        "value": "C0l3PwIhnCmIG3YiAa9jf/uuvZj1ob7lolasnEhDgBE="
+      },
+      "power": "",
```
Comment (mazzy89) [inline misc/deployments/test4.gno.land/genesis.json:25]: @zivkovicmilos which is the voting power we are going to have? I might have missed it.

**Comment by mazzy89** [comment]
FYI @ilgooz

File: misc/deployments/test4.gno.land/genesis.json (hunk 2 above)
Comment (zivkovicmilos) [reply misc/deployments/test4.gno.land/genesis.json:25]: We will all have the same voting power - `1` -- I'll update this in the main PR

File: misc/deployments/test4.gno.land/genesis.json (hunk 1 above)
Comment (zivkovicmilos) [reply misc/deployments/test4.gno.land/genesis.json:35]: This is a good question 👀 

I think the one you've used here is fine:
`<name>-val-<number>`

**Review by zivkovicmilos — APPROVED** [review]
Looking good 💯

**Pull request merged**